tools/aws_lab_setup/roles/common/tasks/RedHat.yml

```diff
@@ -12,9 +12,22 @@
   tags:
     - common
 
-- name: RHEL | Stop and disable firewall
+- name: RHEL | Enable specific firewall ports
+  firewalld:
+    immediate: yes
+    permanent: yes
+    service: "{{ item }}"
+    state: enabled
+  with_items: "{{ firewall_ports }}"
+  notify: restart firewalld
+
+- name: RHEL | Start firewalld
```

Comment:
Still getting the hang of github, but looks like my latest commit to my local branch automatically got added to the PR. Cool, but unexpected, hope it's not made things too messy.